test/optional-validation.test.js

```diff
@@ -308,6 +308,19 @@ describe('optional-validation', function() {
           d.updateAttributes({'name': NEW_NAME}, expectChangeSuccess(done));
         });
       });
+
+      it('returns error when updating id property when forceId is set', function(done) {
+        ModelWithForceId.create({name: 'foo'}, function(err, created) {
+          if (err) return done(err);
+          created.updateAttributes({'id': 123}, function(err, updated) {
+            err.should.be.instanceOf(Error);
+            var msg = 'id property (id) cannot be updated (from ' +
+                       created.id + ' to 123); forceId is set to true';
+            err.message.should.eql(msg);
```

Comment:
err.message.should.eql('`id` cannot be updated from `model.id` to `123` when `forceId` is set to `true`.)